tests/puppeteer/pages/Install/install.js

```diff
@@ -0,0 +1,149 @@
+const CommonPage = require('../commonPage');
+
+module.exports = class INSTALL extends CommonPage {
+  constructor(page) {
+    super(page);
+
+    // Define Step Titles
+    this.firstStepFrTitle = 'Bienvenue sur l\'installateur de PrestaShop 1.7.7.0';
+    this.firstStepEnTitle = 'Welcome to the PrestaShop 1.7.7.0 Installer';
+    this.secondStepEnTitle = 'License Agreements';
+    this.thirdStepEnTitle = 'We are currently checking PrestaShop compatibility with your system environment';
+    this.fourthStepEnTitle = 'Information about your Store';
+    this.fifthStepEnTitle = 'Configure your database by filling out the following fields';
+    this.finalStepEnTitle = 'Your installation is finished!';
+
+    // Selectors for all steps
+    this.nextStepButton = '#btNext';
+
+    // Selectors for step 1
+    this.firstStepPageTitle = 'h2';
+    this.languageSelect = '#langList';
+
+    // Selectors for step 2
+    this.secondStepPageTitle = 'h2#licenses-agreement';
+    this.termsConditionsCheckbox = '#set_license';
+
+    // Selectors for step 3
+    this.thirdStepPageTitle = '#sheet_system h2';
+    this.stepFinishedLeftMenu = '#leftpannel #tabs li.finished:nth-child(3)';
+
+    // Selectors for step 4
+    this.fourthStepPageTitle = '#infosShopBlock h2';
+    this.shopNameInput = '#infosShop';
+    this.countrySelect = '#infosCountry';
+    this.firstNameInput = '#infosFirstname';
+    this.lastNameInput = '#infosName';
+    this.emailInput = '#infosEmail';
+    this.passwordInput = '#infosPassword';
+    this.repeatPasswordInput = '#infosPasswordRepeat';
+
+    // Selectors for step 5
+    this.fifthStepPageTitle = '#dbPart h2';
+    this.dbLoginInput = '#dbLogin';
+    this.dbPasswordInput = '#dbPassword';
+    this.tablePrefixInput = '#db_prefix';
+    this.testDbConnectionButton = '#btTestDB';
+    this.createDbButton = '#btCreateDB';
+    this.dbResultCheckOk = '#dbResultCheck.okBlock';
+
+    // Selectors for Final step
+    this.finalStepPageTitle = '#install_process_success h2';
+    this.installationProgressBar = '#install_process_form #progress_bar .installing';
+    this.discoverFoButton = '#foBlock';
+
+    // Selectors in FO
+    this.FOLogo = '#_desktop_logo';
+  }
+
+  /**
+   * To check each step title
+   * @param selector, where to get actual title
+   * @param pageTitle, expected title
+   */
+  async checkStepTitle(selector, pageTitle) {
+    await this.page.screenshot({path: 'test.png'});
```

Comment:
Maybe don't hard-code the name `test.png` ? It will be overwritten every time the method is called

Reply:
I added it to test.
Deleted thanks